eclipse-scout-core/src/form/fields/slider/SliderField.ts

```diff
@@ -27,31 +55,115 @@ export class SliderField extends ValueField<number> {
+    this.addStatus();
+  }
+
+  protected override _remove() {
+    super._remove();
+    this.slider.remove();
+    this.$valueLabel = null;
+    this.$fieldHtmlComp = null;
+  }
+
+  protected override _renderProperties() {
+    super._renderProperties();
+    this._renderValueEditable();
+  }
+
+  protected override _renderEnabled() {
+    super._renderEnabled();
+    this.slider.setEnabled(this.enabledComputed);
+  }
+
+  protected override _renderDisplayText() {
+    super._renderDisplayText();
+    this.syncDisplayTextToSlider(this.displayText);
+    this.$valueLabel.text(this.displayText);
+  }
+
+  protected _renderValueEditable() {
+    this.$field.setVisible(this.valueEditable);
+    this.$valueLabel.setVisible(!this.valueEditable);
+  }
+
+  _onSliderValueChanged(event: PropertyChangeEvent) {
+    if (!this.syncingValue) {
+      this.setValue(event.newValue);
+    }
+  }
+
+  protected override _setMinValue(minValue: number) {
+    super._setMinValue(minValue);
+    if (this.slider) {
+      this.slider.setMinValue(minValue);
+    }
+  }
+
+  protected override _setMaxValue(maxValue: number) {
+    super._setMaxValue(maxValue);
+    if (this.slider) {
+      this.slider.setMaxValue(maxValue);
+    }
+  }
+
+  setStep(step: number) {
+    this.setProperty('step', step);
+  }
+
+  _setStep(step: number) {
+    this._setProperty('step', step);
+    if (!objects.isNullOrUndefined(step)) {
+      this.slider.setStep(step);
+    }
+  }
+
+  setTabbable(tabbable: boolean) {
+    this.setProperty('tabbable', tabbable);
   }
 
-  protected _renderSlider() {
-    this.slider.render();
-    this.addField(this.slider.$container);
+  _setTabbable(tabbable: boolean) {
+    this._setProperty('tabbable', tabbable);
+    if (this.slider) {
+      this.slider.setTabbable(tabbable);
+    }
   }
 
-  protected override _readDisplayText(): string {
-    // Use the inner slider's value as display text, as the user cannot enter the value manually.
-    // This value is already guaranteed to be a valid number (see Slider.js, _onValueChange). We
-    // convert it to a string to match the expected data type for a display text.
-    return String(this.slider.value);
+  setValueEditable(valueEditable: boolean) {
+    this.setProperty('valueEditable', valueEditable);
   }
 
-  protected override _parseValue(displayText: string): number {
-    // Convert display text back to number
-    return Number(displayText);
+  protected override _setValue(value: number | string) {
+    super._setValue(scout.nvl(value, this.minValue));
   }
 
-  override setValue(value: number) {
-    this.slider.setValue(value);
+  protected override _onFieldInput() {
+    this.syncDisplayTextToSlider(this._readDisplayText());
   }
 
-  getValue(): number {
-    return this.slider.value;
+  protected syncDisplayTextToSlider(displayText: string) {
+    try {
+      this.syncingValue = true;
+      this.slider.setValue(this._parseValue(displayText));
+    } catch (e: any) { /* empty */
```

Comment:
```suggestion
    } catch (error) {
      // nop
```

- No need to declare the error type, since we are not using it
- Code does not work faster if we use shorter variable names :-)
- Avoid /* comments */, it is perfectly fine to use a full line for a line comment